api/types/types.go

```diff
@@ -512,12 +512,13 @@ type DiskUsage struct {
 // ImagesPruneConfig contains the configuration for Engine API:
 // POST "/images/prune"
 type ImagesPruneConfig struct {
-	DanglingOnly bool
+	Filters filters.Args
```

Comment:
Don't forget to update the swagger.yml do match this change; https://github.com/docker/docker/blob/22ba220e864d6d30ce1ed56a5461b96dbade844f/api/swagger.yaml#L4830-L4868